src/printer/tvmscript_printer.cc

```diff
@@ -119,6 +119,10 @@ class TVMScriptPrinter : public StmtFunctor<Doc(const Stmt&)>,
   std::unordered_map<Buffer, Doc, ObjectPtrHash, ObjectPtrEqual> memo_buf_;
   /*! \brief Map from Buffer to Declaration Doc */
   std::unordered_map<Buffer, Doc, ObjectPtrHash, ObjectPtrEqual> memo_buf_decl_;
+  /*! \brief Map from SparseBuffer to Doc */
```

Comment:
Is `Doc` a data structure for printing?

Reply:
Yes exactly. It's a class in TVMScript printer.